disperser/controller/dispatcher.go

```diff
@@ -4,11 +4,12 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"github.com/prometheus/client_golang/prometheus"
 	"math"
 	"time"
 
-	"github.com/Layr-Labs/eigenda/api/clients"
+	"github.com/prometheus/client_golang/prometheus"
+
+	clients "github.com/Layr-Labs/eigenda/api/clients/v2"
```

Comment:
I don't think this would be a big refactor to do just within the `api` package. 
Yes, it would be more difficult to do this in other packages because there's a lot of sharing

Reply:
I think it would be very nice to do, but don't have enough context on everything in the api package, and whether this refactor would make sense with the rest of the code in this repo.